docs/user-guides/userguide_linux.md

```diff
@@ -57,45 +59,65 @@ The Mumble Server is either started [automatically](#auto-start) or [manually](#
 On many distributions the Mumble Server is started automatically on system boot.
 
 You can also disable this and start the Server manually.
-Most distributions use `systemd` now, so search in the systemd documentation on how to disable the Mumble Server service.
+Most distributions use `systemd` now, so search in the systemd documentation on how to disable the Mumble Server service (e.g. `murmur.service`).
+
+Next Steps: 
+
+1. Set the SuperUser (Server Admin) password:
+
+```
+murmurd -ini /etc/murmur.ini -supw PASSWORD
+```
+
+2. Restart the Server:
+If your system uses `systemd`, use e.g.:
+
+```
+systemctl restart murmur.service
+```
+
+3. Login & administrate the Server:
+See [Server Configuration Guide](server_config_guide.md#administration-with-mumble-client) for details.
 
 #### Manual Start
 
-The Mumble Server should be run from the command line, so start a terminal.
-(*If the `Static Linux Server` is installed or the binary is not linked, you have to change directory to wherever the Mumble Server is installed, first.*)
+The Mumble Server should be run from a terminal.
+(*If the `Static Linux Server` is installed or the binary is not linked, you have to change the directory to wherever the Mumble Server is installed, first.*)
 
-Run the Mumble Server with:
+<!-- Will the binary name change in 1.4? -->
```

Comment:
We currently don't have plans for that.

Reply:
I remembered a discussion about the naming and now it is not planned anymore?
#4046

Reply:
It never was planned to rename the executable (afaik). This discussion was more referring to how the server should be referred to (in package archives).
@davidebeatrici did I misunderstand this?

Reply:
Update: Binary names will probably be changed, see #4046.